package/Resources/Translations/zh-Hans/Translation.ini

```
@@ -0,0 +1,1704 @@
+Client:Main:Unknown=未知; Unknown
+Client:Main:UnknownPing=未知; Unknown
+Client:Main:UnknownPlayer=未知玩家; Unknown player
+Client:Main:UnknownSide=未知; Unknown
+Client:Main:UnlockGame=解锁游戏; Unlock Game
+Client:Main:UnpackingArchive=正在解压存档包; Unpacking archive
+Client:Main:UpdateAvailable=有可用更新。; An update is available.
+Client:Main:UpdateAvailableClickToInstall=有可用更新，点击安装。; An update is available, click to install.
+Client:Main:UpdateAvailableManualDownloadRequired=有可用更新。需要手动下载和安装。; An update is available. Manual download & installation required.
+Client:Main:UpdateAvailableText=有可用更新。您想现在执行更新吗？; An update is available. Do you want to perform the update now?
+Client:Main:UpdateAvailableTitle=有可用更新; Update available
+Client:Main:UpdateCancelledClickToRetry=更新已取消。点击重试。; The update was cancelled. Click to retry.
+Client:Main:UpdateFailedClickToRetry=更新失败！点击重试。; Updating failed! Click to retry.
+Client:Main:UpdateFailedText=更新时发生错误。返回的错误是：{0}@@如果您已连接到互联网且防火墙没有阻止 {1}，并且问题可以复现，请联系我们 {2} 以获取支持。; An error occured while updating. Returned error was: {0}@@If you are connected to the Internet and your firewall isn't blocking@{1}, and the issue is reproducible, contact us at@{2} for support.
+Client:Main:UpdateFailedTitle=更新失败; Update failed
+Client:Main:UpdateMapToDBFailed=上传地图 {0} 到 CnCNet 地图数据库失败。; Uploading map {0} to the CnCNet map database failed.
+Client:Main:UpdateMapToDBPrompt={0} 的本地安装中没有地图“{1}”。正在尝试将地图上传到 CnCNet 地图数据库。; {0} doesn't have the map '{1}' on their local installation. Attempting to upload the map to the CnCNet map database.
+Client:Main:UpdateMapToDBSuccess=上传地图 {0} 到 CnCNet 地图数据库完成。; Uploading map {0} to the CnCNet map database complete.
+Client:Main:UpdateSizeKB=更新大小为 {0} KB。; The size of the update is {0} KB.
+Client:Main:UpdateSizeMB=更新大小为 {0} MB。; The size of the update is {0} MB.
+Client:Main:UpdateStatusConnecting=连接中; Connecting
+Client:Main:UpdateSuccess={0} 已成功更新到 v.{1}; {0} was succesfully updated to v.{1}
+Client:Main:UpdateVersionPleaseWait=请等待 {0} 更新到版本 {1}。@更新完成后，此窗口将自动关闭。@@下载更新后，客户端也可能重新启动。; Please wait while {0} is updated to version {1}.@This window will automatically close once the update is complete.@@The client may also restart after the update has been downloaded.
+Client:Main:Updating=更新中...; Updating...
+Client:Main:UserNotAvailable=用户当前不可用！; User is not currently available!
+Client:Main:UserNotInGame={0} 不在游戏中！; {0} is not in a game!
+Client:Main:VersionAvailable=版本 {0} 可供下载。@您希望安装它吗？; Version {0} is available for download.@Do you wish to install it?
+Client:Main:ViewChangeLog=查看更新日志; View Changelog
+Client:Main:YouMustReplaceMap=您需要更换地图，否则一些玩家将无法参与此比赛。; You need to change the map or some players won't be able to participate in this match.
+Client:Main:YouNeedChangeMapForPlayer=您需要更换地图，否则 {0} 将无法参与此比赛。; You need to change the map or {0} won't be able to participate in this match.
+Client:Main:YourColor=您的颜色：; YOUR COLOR:
+Client:Main:YouWereKicked=你被踢出游戏了！; You were kicked from the game!
```

Make it consistent with other translations

```suggestion
Client:Main:YouWereKicked=您被踢出游戏了！; You were kicked from the game!
```